zubhub_frontend/zubhub/src/views/create_project/script.js

```diff
@@ -693,21 +693,8 @@ export const validationSchema = Yup.object().shape({
     //       ? false
     //       : true;
     //   }),
-    materials_used: Yup.string()
-        .max(10000, 'max')
-        .test('empty', 'required', value => {
-            let is_empty = true;
-
-            value &&
-                value.split(',').forEach(material => {
-                    if (material) {
-                        is_empty = false;
-                    }
-                });
-
-            return !is_empty;
-        }),
-    category: Yup.string(),
+    materials_used: Yup.array().required('required'),
```

Comment:
No idea why we keep removing  validations instead of getting them to work properly

Reply:
If we are going to work on this then we should fix it completely